xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -3044,11 +3072,6 @@ public void streamClosedAndRetryReschedulesAllResourceFetchTimer() {
     assertThat(cdsRespTimeoutTask.getDelay(TimeUnit.SECONDS))
         .isEqualTo(XdsClientImpl.INITIAL_RESOURCE_FETCH_TIMEOUT_SEC);
 
-    fakeClock.forwardTime(XdsClientImpl.INITIAL_RESOURCE_FETCH_TIMEOUT_SEC, TimeUnit.SECONDS);
-    ArgumentCaptor<Status> statusCaptor = ArgumentCaptor.forClass(null);
-    verify(clusterWatcher).onError(statusCaptor.capture());
-    assertThat(statusCaptor.getValue().getCode()).isEqualTo(Code.NOT_FOUND);
```

Comment:
You can `verify(clusterWatcher, atLeastOnce()).onError(statusCaptor.capture());` and still verify the last status captured is NOT_FOUND.

Reply:
In this test, I want to explicitly keep tracking of operations happening step by step. So it would be better to explicitly count the number of invocations.

Reply:
But why did you remove an existing behavior test?

Reply:
I mean, these checks are not critical to this test case (i.e.,  not what this test case is targeting fo). So I clean them up. See the updated comments for my thought.

Reply:
OKay, not big deal, as the removed behavior is covered in other tests.